datanode/gateway/graphql/schema.graphql

```diff
@@ -256,6 +256,8 @@ type MarketData {
   marketGrowth: String!
   "The last traded price (an unsigned integer)"
   lastTradedPrice: String!
+  "The current funding rate. This applies only to a perpetual market"
+  fundingRate: String!
```

Comment:
Firstly, what will the value be if this market is not a perp. I would expect a nullable field here?

Secondly, will there be another pr to have this field added to ObservableMarketData too? This is key for frontend requirements